packages/pieces/community/pdf/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@activepieces/piece-pdf",
-  "version": "0.2.17",
+  "version": "0.3.0",
```

Comment:
You can bump the minor version (0.2.18) since adding an action is not a breaking change